backend/api/routes.py

```diff
@@ -1,12 +1,107 @@
+import logging
+import os
+import sys
+
+sys.path.insert(0, "../../backend")
+
+from api.visualization import generate_chords
```

Comment:
Should not be in the api package imo

Reply:
I can omit this but without it, for some reason, api is not recognized as a module in VSCode.

Reply:
Oh that's an issue too then but I meant the package in line 7 - I'd keep the api package for REST functions

Reply:
Fixed it